src/screens/ContactUs.js

```diff
@@ -33,20 +33,20 @@ const ScreensContactUs = () => {
       <div id="page">
         <main id="main-content" className="content" tabIndex="0">
           <Breadcrumbs items={breadCrumbs} />
-          <p>
+          <p data-testid="location-text">
             Northwestern University Libraries' Repository and Digital Curation
             department is located in the main library on our Evanston campus.
             The department is responsible for digitization of library
             collections, describing content, and creating the software that runs
             this website.
           </p>
 
-          <p>
+          <p data-testid="questions-text">
             Questions about digitization, description of items, or software can
             be directed here. <Mailto email="repository@northwestern.edu" />
           </p>
 
-          <p>
+          <p data-testid="content-text">
             The content on the site comes from multiple library collections,
             departments and units. Please visit{" "}
             <a
```

Comment:
Let's remove the context of the paragraph's content from the `data-testid` values.  This content will adjust over time, and `data-testid="location-text"` might change and be about say, history rather than location.   

I think it's safe to just check that some text appears in at least one paragraph on the page.